packages/lit-dev-tools-cjs/package.json

```diff
@@ -6,9 +6,34 @@
   "author": "Google LLC",
   "license": "BSD-3-Clause",
   "scripts": {
-    "build": "npm run build:ts",
-    "build:ts": "../../node_modules/.bin/tsc",
-    "test": "uvu ./lib \".spec.js$\""
+    "build": "wireit",
+    "build:ts": "wireit",
+    "test": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:ts"
+      ]
+    },
+    "build:ts": {
+      "files": [
+        "src/**",
+        "package.json"
```

Comment:
We do have a tsbuildinfo file declared, and we set incremental/composite here: https://github.com/lit/lit.dev/blob/c7ca68dcf85519b597efbc487140718e8ea72b65/packages/lit-dev-tools-cjs/tsconfig.json#L7

So this might be doing incremental build.

In which case, you need to include the tsbuildinfo file, and set `"clean": "if-file-deleted"`

Reply:
+1, done

The tsbuildinfo file is included, as it's emitted into the lib directory